### ISSUE TITLE
Killshot?

### ISSUE THREAD

**Issue by 4stax** [danielkrupinski/Osiris#514]
Does killshot make the triggerbot/aimbot only function if a kill would be gained from the damage?
Could someone help clarify? Thanks!

**Comment by Bywnb** [comment]
i also dont know killshot how to use

**Comment by rosesware** [comment]
Read the name, holy shit.

**Comment by danielkrupinski** [comment]
Yes, it makes triggerbot fire only if shot will deal damage higher or equal enemy's health. In aimbot it does the same but only for autowall.
